src/main/scala/no/gdl/bookapi/integration/ImageApiClient.scala

```diff
@@ -43,8 +43,10 @@ trait ImageApiClient {
     private def doRequest(httpRequest: HttpRequest): Option[ImageMetaInformation] = {
       gdlClient.fetch[ImageMetaInformation](httpRequest) match {
         case Success(metaInfo) => Some(metaInfo)
-        case Failure(hre: HttpRequestException) => if (hre.is404) None else throw hre
-        case Failure(ex: Throwable) => throw ex
+        case Failure(ex: Throwable) =>  {
+          logger.error(s"Got ${ex.getClass.getSimpleName} when calling ${httpRequest.url}: ${ex.getMessage}")
+          None
```

Comment:
Hmm....Dette blir nok kanskje litt for enkelt.
Er flere veier inn til denne metoden, som krever forskjellig håndtering av at bilde ikke blir funnet.

Se `ConverterService.toApiBookInternal`:
```
toApiCoverPhoto(translation.coverphoto)

def toApiCoverPhoto(imageIdOpt: Option[Long]): Option[api.CoverPhoto] = {
      imageIdOpt.flatMap(imageId =>
        imageApiClient.imageMetaWithId(imageId))
        .map(imageMeta => {
          val large = imageMeta.imageUrl
          val small = s"$large?width=200"
          api.CoverPhoto(large, small)
        })
    }
```
Her gir det mening å ha en None

Se `ContentConverter.toApiContent`:
```
val url = imageApiClient.imageUrlFor(nodeId.toLong) match {
   case Some(x) => x
   case None => throw new NotFoundException(s"Could not find image for id $nodeId")
}
``` 
Denne funksjonaliteten må også endres, ellers er vi like langt. Denne burde muligens kun returnere en tag som sier at her mangler et bilde.

Se `EPubService.buildEPubFor`:
```
imageApiClient.downloadImage(coverPhotoId) match {
   case Failure(ex) => throw ex
   case Success(downloadedImage) =>
    book.addCoverImage(
      downloadedImage.bytes,
      downloadedImage.metaInformation.contentType,
      downloadedImage.metaInformation.imageUrl.pathParts.last.part)
}
```
Her gir det mening å kaste en exception, for at vi ikke skal drive og cache (i cloudfront) en bok som ikke har bilder i seg..

Reply:
I metoden downloadImage returneres Failure dersom imageMetaWithId returnerer None, så epub-funksjonaliteten vil fremdeles fungere.

Så det vi treng å endre er i ContentConverter som kan sette url til ei dummy-adresse.